src/Data/Rational/Properties.agda

```diff
@@ -64,6 +63,7 @@ open import Relation.Nullary.Decidable.Core as Dec
   using (yes; no; recompute; map′; _×-dec_)
 open import Relation.Nullary.Negation.Core using (¬_; contradiction)
 
+open import Algebra.Consequences.Propositional {A = ℚ}
```

Comment:
Why do we now need to import it with the type specifically applied? Is this a breaking change?

Reply:
And in fact, *this* one is the unneeded `import`!